File: src/main/java/me/reckter/telegram/requests/UpdateMessageRequest.java
package me.reckter.telegram.requests;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Created by hannes on 18/06/2016.
 */
public class UpdateMessageRequest extends MessageRequest {


    @JsonProperty("message_id")
    public long messageId;

    public long getMessageId() {
        return messageId;
    }

    public void setMessageId(long messageId) {
        this.messageId = messageId;
    }
}

File: src/main/java/me/reckter/telegram/requests/UpdateMessageRequest.kt
package me.reckter.telegram.requests

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * Created by hannes on 18/06/2016.
 */
class UpdateMessageRequest : MessageRequest() {


    @JsonProperty("message_id")
    var messageId: Long? = null

    @JsonProperty("inline_message_id")
    var inlineMessageId: String? = null
}
